_posts/meat/2020-11-01-cassoulet.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Cassoulet
+category: meat
+tags: recipeish
+---
+
+We've been on a cassoulet kick this summer in the foggy Inner Richmond. The first few times we made it with duck confit pieces. This last time, we had run out of duck confit on hand, but still had some of the [jelly that sinks at the bottom of duck fat)](https://www.reddit.com/r/AskCulinary/comments/6g2klj/the_gelatin_of_duck_fat/) when you render it yourself, so we used that instead. 
```

Comment:
> duck confit pieces

duck confit ~pieces~. 'Pieces' here is redundant.

Reply:
I think the 'jelly' here is actually called 'aspic', but maybe calling it that would sound pretentious. 'duck gelatin'?